utils/config.mlp

```diff
@@ -122,8 +122,8 @@ let max_tag = 243
 let lazy_tag = 246
 
 let max_young_wosize = 256
-let stack_threshold = 16 (* see runtime/caml/config.h *)
-let stack_safety_margin = 60
+let stack_threshold = 32 (* see runtime/caml/config.h *)
+let stack_safety_margin = 6
```

Comment:
It could be nice for future yous to include a reference for the choice of stack_safety_marging ; this PR or the comment by Jacques-Henry on the older PR.